app/code/Magento/Customer/Model/Metadata/Form/AbstractData.php

```diff
@@ -98,6 +105,7 @@ public function __construct(
         \Psr\Log\LoggerInterface $logger,
         \Magento\Customer\Api\Data\AttributeMetadataInterface $attribute,
         \Magento\Framework\Locale\ResolverInterface $localeResolver,
+        \Magento\Framework\Filesystem\Driver\Http $http,
```

Comment:
Please make this parameter optional with fallback to object manger due to backward compatibility reasons